modules/nf-core/taxpasta/merge/meta.yml

```diff
@@ -0,0 +1,47 @@
+name: "taxpasta_merge"
+description: Standardise and merge two or more taxonomic profiles into a single table
+keywords:
+  - standardise
+  - merge
+tools:
+  - "taxpasta":
+      description: "TAXonomic Profile Aggregation and STAndardisation"
+      homepage: "https://taxpasta.readthedocs.io/"
+      documentation: "https://taxpasta.readthedocs.io/"
+      tool_dev_url: "https://github.com/taxprofiler/taxpasta"
+      doi: ""
+      licence: "['Apache-2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - profile:
+      type: file
+      description: profiler output file
+      pattern: "*.{txt,tsv,out}{.gz}"
+  - samplesheet:
+      type: file
+      description: Directory containing at a minimum nodes.dmp and names.dmp files
+      pattern: "*.{tsv,csv,ods,xlsx,arrow,parquet}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - profiles:
+      type: file
+      description: The desired output file
```

Comment:
Needs to be more descriptive, see standardise, and also mention it's containing all samples in one file